src/ngx_http_lua_headers.c

```diff
@@ -485,7 +485,17 @@ ngx_http_lua_ngx_resp_get_headers(lua_State *L)
     {
         extra++;
         lua_pushliteral(L, "content-length");
-        lua_pushfstring(L, "%d", (int) r->headers_out.content_length_n);
+        if (r->headers_out.content_length_n > NGX_MAX_INT32_VALUE) {
+            u_char *p = ngx_palloc(r->pool, NGX_OFF_T_LEN);
```

Comment:
Better to move the declaration of `p` to the top of function, to make the code style consistent.